src/main/js/components/timeline/TweetBlock.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import "../../../css/components/timeline/TweetBlock.scss";
+
+export default class TweetBlock extends React.Component {
+
+	static formatTweet(tweet) {
```

Comment:
be careful with what you are you doing with your data here as you want to be sure of what you are doing with your data...please explain

Reply:
I see that if the tweet prop is undefined it could cause an error. I've fixed this now, but was there more you wanted me to explain here? I'm using this function to avoid errors and format the date.